README.md

```diff
@@ -153,6 +153,22 @@ If you’re fixing large of amounts of previously unformatted code, consider tem
       }
       ```
 
+    - `forceFormatExtracted`: Enables formatting of extracted fragements, (default: `false`).
+      ESLint supports processors that let you extract and lint JS fragments within a non-JS language, for example using `eslint-plugin-markdown` to extract and lint a code block inside a Markdown file. Usually, you don't want to have the fragement itself formatted by Prettier as the whole file
+      might already be formatted anyway. This is why `eslint-plugin-prettier` ignores such fragements by default. However, for cases you want to have such extracted fragements formatted by Prettier you can use this option to force it.
```

Comment:
> However, for cases you want to have such extracted fragements formatted by Prettier you can use this option to force it.

So, the case is, you want to format js in markdown via prettier, but not the markdown file itself? cc @BPScott

Reply:
Yeah, sort of. Actually I'd like to be able to apply a different set of Prettier rules to code blocks in Markdown files resp. be able to individually handle those blocks (for example disabling Prettier for JS code blocks, while formatting other code blocks like JSON). This can be achieved via ESLint rules on Markdown files processed with `eslint-plugin-markdown`. This set-up is working fine, but only if I'm able to disable the default behavior of `eslint-plugin-prettier` with this newly introduced option to not skip those code blocks.

Reply:
Commented down below. I think there's ways of doing this without introducing a new option.